CHANGELOG.asciidoc

```diff
@@ -29,6 +29,7 @@ This release also includes changes from <<release-3-3-8, 3.3.8>>.
 * Modified `NumberHelper` to better ignore `Double.NaN` in `min()` and `max()` comparisons.
 * Bump to Netty 4.1.36.
 * Bump to Groovy 2.5.7.
+* TINKERPOP-2243 Added `userAgent` to RequestOptions. Gremlin Console sends `Gremlin Console` as the `userAgent`.
```

Comment:
nit: No need to reference the JIRA here. that can be removed. we capture all the JIRAs closed on this version on release.